docs/_templates/2025/menu-desktop.html

```diff
@@ -65,7 +65,9 @@
                 <li><a href="{{ pathto('conf/'+shortcode+'/'+year_str+'/visiting') }}">Visiting {{ city }}</a></li>
                 <li><a href="{{ pathto('conf/'+shortcode+'/'+year_str+'/attendee-guide') }}">Attendee Guide</a></li>
                 <li><a href="{{ pathto('conf/'+shortcode+'/'+year_str+'/health') }}">Health and Safety Policy</a></li>
+                <li><a href="{{ pathto('conf/'+shortcode+'/'+year_str+'/speaker-info') }}">Speaker Info</a></li>
```

Comment:
Not 100% sure we want this public, but I guess it's useful for lightning talks as well? I could see it being a "hidden" page, but also fine with it being public. 

I'd put it above the COC though -- it's very low usefulness for an attendee vs. the Virtual attendence info.

Reply:
Fair point, I'll remove and link it in all speaker emails.